package.json

```diff
@@ -19,7 +19,6 @@
     "tsc": "tsc --noEmit --skipLibCheck",
     "watch:server": "nodemon --inspect ./build/server.js",
     "watch": "webpack --watch --config webpack.development.js",
-    "dev": "stmux -M -e ERROR -m beep -- [ [ \"yarn watch\" .. \"yarn watch:server\" ] ]",
```

Comment:
removing `yarn dev` would mean the `make dev` command wouldn't work

either `yarn dev`  or `make dev` should become `yarn watch:server & yarn watch & wait`

Reply:
oh hm, I thought I had changed that - thanks for catching

Reply:
fixed 👍